extensions/vscode/package.json

```diff
@@ -182,6 +182,12 @@
         "title": "Toggle PearAI Inventory",
         "group": "PearAI"
       },
+      {
+        "command": "pearai.startOnboarding",
+        "category": "PearAI Developer",
+        "title": "Start PearAI Onboarding",
+        "group": "PearAI"
+      },
```

Comment:
my computer was quite slow so sometimes, on reset, the onboarding flow wouldn't popup (I think there was a 10 second timeout or something like that). Adding this allowed me to reliably trigger it